test/lib/actions/ConfigAction.test.ts

```diff
@@ -264,12 +261,11 @@ describe('ConfigAction tests', () => {
 				const output = await runActionAndGetDisplayedConfig(dependencies, ['all']);
 
 				// ==== ASSERTIONS ====
-				const defaultConfig = CodeAnalyzerConfig.withDefaults();
 				const goldFileContents = (await readGoldFile(path.join(PATH_TO_COMPARISON_DIR, 'derivables-as-non-defaults', `${prop}.yml.goldfile`)))
 					.replace('__DUMMY_CONFIG_ROOT__', parentOfCurrentDirectory)
 					.replace('__DUMMY_LOG_FOLDER__', parentOfCurrentDirectory)
-					.replace('__DUMMY_DEFAULT_CONFIG_ROOT__', JSON.stringify(defaultConfig.getConfigRoot()))
-					.replace('__DUMMY_DEFAULT_LOG_FOLDER__', JSON.stringify(defaultConfig.getLogFolder()))
+					.replace('__DUMMY_DEFAULT_CONFIG_ROOT__', JSON.stringify(null))
```

Comment:
`JSON.stringify(null)` feels weird to me.

Reply:
Yeah... I was a little lazy here. I could have just put 'null'.

Reply:
Fixed.